### PR TITLE
bugfix/cplp 2064/idp cleanup

### PR DESCRIPTION
**Pull request by oyo**

### CODE REVIEW

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]
idp.json for german translations to be added

File: cx-portal/src/components/overlays/CXValidatingOverlay/ValidatingInput.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  /*
+  useEffect(() => {
+    console.log('useEffect')
+    if (value) {
+      setData(value)
+      immediateValidation(value)
+    }
+  }, [value, immediateValidation])
+  */
+
```
Comment (lavanya-bmw) [inline cx-portal/src/components/overlays/CXValidatingOverlay/ValidatingInput.tsx:92]: Commented code can be removed?

File: cx-portal/src/components/overlays/UpdateIDP/UpdateIDPContent.tsx [hunk 2]
```diff
@@ -185,24 +115,26 @@ const UpdateIDPForm = ({
...
         <ValidatingInput
           name="metadataUrl"
           label={t('field.metadata.name')}
-          validate={isWellknownMetadata}
-          tooltipMessage={t('field.metadata.hint')}
-          helperText={t('field.metadata.hint')}
+          validate={(expr) => {
+            console.log(expr)
+            return isWellknownMetadata(expr)
+          }}
```
Comment (lavanya-bmw) [inline cx-portal/src/components/overlays/UpdateIDP/UpdateIDPContent.tsx:125]: Console can be removed and validate can be: validate={(expr) => isWellknownMetadata(expr) }

**Review by lavanya-bmw — CHANGES_REQUESTED** [review]

**Comment by lavanya-bmw** [comment]
> idp.json for german translations to be added

Added

**Review by lavanya-bmw — DISMISSED** [review]
As discussed, Martin will work on duplication of code later

**Comment by jjeroch** [comment]
Findings are all added in ticket https://jira.catena-x.net/browse/CPLP-2465

**Comment by manojava-gk** [comment]
Just to avoid duplication. Created new MyForm component.

**Review by lavanya-bmw — APPROVED** [review]

**Pull request merged**